profiles/api/oauth2.py

```diff
@@ -136,4 +138,14 @@ def token() -> Response:
     filtered_json_data = {your_key: json_data[your_key] for your_key in
                           ['access_token', 'expires_in', 'name', 'orcid', 'token_type']}
 
+    try:
+        profile = Profile.query.filter_by(orcid=filtered_json_data['orcid']).one()
+        profile.name = filtered_json_data['name']
+    except NoResultFound:
+        profile = Profile(filtered_json_data['name'], filtered_json_data['orcid'])
```

Comment:
we may need some sanitization, but the risk is mangling horribly someone's name. Welcome to eLife `dÃ©jÃn`!

Reply:
I will look for a dataset of names to check UTF-8 and possible conversions throughout the stack

Reply:
Always useful. 👍

Reply:
All popular datasets are US-centric, probably will generate random UTF-8 data... But need to check what ORCID returns first.